lighthouse-core/audits/unused-css-rules.js

```diff
@@ -79,6 +89,42 @@ class UnusedCSSRules extends Audit {
   }
 
   /**
+   * Trims stylesheet content down to the first rule-set definition.
+   * @param {string} content
+   * @return {string}
+   */
+  static determineContentPreview(content) {
+    let preview = content
+        .slice(0, PREVIEW_LENGTH * 5)
+        .replace(/( {2,}|\t)+/g, '  ') // remove leading indentation if present
+        .replace(/\n\s+}/g, '\n}') // completely remove indentation of closing braces
+        .trim(); // trim the leading whitespace
+
+    if (preview.length > PREVIEW_LENGTH) {
+      const firstRuleStart = preview.indexOf('{');
+      const firstRuleEnd = preview.indexOf('}');
+
+      if (firstRuleStart === -1 || firstRuleEnd === -1
```

Comment:
Also I would love to just totally kill this at some point in the future by attributing where the inline sheet came from and/or maybe showing a few of the selectors